Update available-enterprise-tier-only.md (#3948)
fishtown-analytics/docs.getdbt.com: website/snippets/available-enterprise-tier-only.md

Connecting an Azure DevOps cloud account is available for organizations using the dbt Cloud Enterprise tier. 

> Azure DevOps Server (on-premises) is not supported on dbt Cloud's native Azure DevOps integration. Please use [Import a project by git URL](/docs/cloud/git/import-a-project-by-git-url) instead to connect to Azure DevOps Server.

Reviewer: ```suggestion
dbt Cloud's native Azure DevOps integration does not support Azure DevOps Server (on-premise). Instead, you can [import a project by git URL](/docs/cloud/git/import-a-project-by-git-url) to connect to an Azure DevOps Server.
```
